eng/Versions.props

```diff
@@ -11,6 +11,10 @@
     <UsingToolNetFrameworkReferenceAssemblies>true</UsingToolNetFrameworkReferenceAssemblies>
     <UsingToolNuGetRepack>true</UsingToolNuGetRepack>
     <!-- Dependencies -->
+    <MicrosoftSourceBuildIntermediatesourcebuildreferencepackagesPackageVersion>8.0.0-alpha.1.22616.1</MicrosoftSourceBuildIntermediatesourcebuildreferencepackagesPackageVersion>
```

Comment:
Curious how you created these?  Is there tooling that will add these and the version details dependency entry or were these created by hand?

Reply:
Darc created these through `darc add-dependency` + updates the versions once `darc update-dependency` is called (guess the same is done if the update comes from a PR)

Reply:
Thanks - I've always added these by hand.  It's good to know.

Reply:
`MicrosoftSourceBuildIntermediatesourcebuildreferencepackagesPackageVersion` this is hard to read, can it be capitalized correctly?

Reply:
It is created / updated by tooling (`darc`), not sure it will work with a different definition

Reply:
I think it should work with any capitalization.

Reply:
Or maybe it can just be removed from the `Versions.props` completely?

Reply:
Yep, not needed in Versions.props